eclair-core/src/main/scala/fr/acinq/eclair/channel/Commitments.scala

```diff
@@ -310,7 +311,7 @@ object Commitments {
     (commitments1, fee)
   }
 
-  def receiveFee(commitments: Commitments, fee: UpdateFee, maxFeerateMismatch: Double): Commitments = {
+  def receiveFee(feeTargets: FeeTargets, commitments: Commitments, fee: UpdateFee, maxFeerateMismatch: Double, feeEstimator: FeeEstimator): Commitments = {
```

Comment:
commitments should be the first parameter

Reply:
Done in f0649dd